bots/pr/src/main/java/org/openjdk/skara/bots/pr/LimitedCensusInstance.java

```diff
@@ -70,12 +70,9 @@ private static Namespace namespace(Census census, String hostNamespace) {
 
     private static Optional<JCheckConfiguration> configuration(HostedRepository remoteRepo, String name, String ref) {
         Optional<List<String>> conf = Optional.empty();
-        try {
-            conf = Optional.of(Arrays.stream(remoteRepo.fileContents(name, ref).split("\n")).toList());
-        } catch (UncheckedRestException e) {
-            if (e.getStatusCode() != 404) {
-                throw e;
-            }
+        var contents = remoteRepo.fileContents(name, ref);
+        if (contents.isPresent()) {
+            conf = Optional.of(Arrays.stream(contents.get().split("\n")).toList());
         }
         return conf.map(JCheckConfiguration::parse);
```

Comment:
This seems a bit awkward. I think something like this should do it (untested and likely not completely right):
```
return remoteRepo.fileContents(name, ref).map(contents -> JCheckConfiguration.parse(Arrays.stream(contents.split("\n")).toList()));
```